From e72dc01c85fb6baa8e065660bda6ec92e21bee49 Mon Sep 17 00:00:00 2001
From: "Brian C. Lane" <bcl@redhat.com>
Date: Mon, 12 Jul 2021 15:29:29 -0700
Subject: [PATCH 20/24] tests/t3000: Change dev_size_mb to 267

FAT32 needs a minimum partition size of 256MB so this is as small as we
can make it.

This is backed by memory, so using more than is needed limits the size
of the system it can run on.
---
 tests/t3000-resize-fs.sh | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/tests/t3000-resize-fs.sh b/tests/t3000-resize-fs.sh
index 64b038d..1f2e46e 100755
--- a/tests/t3000-resize-fs.sh
+++ b/tests/t3000-resize-fs.sh
@@ -29,8 +29,8 @@ start=63s
 default_end=546147s
     new_end=530144s
 
-# create memory-backed device
-scsi_debug_setup_ dev_size_mb=550 > dev-name ||
+# create memory-backed device. Must be > 256MB+8MB
+scsi_debug_setup_ dev_size_mb=267 > dev-name ||
   skip_ 'failed to create scsi_debug device'
 dev=$(cat dev-name)
 
-- 
2.31.1

